PlatformIO/src/StateMachine.hpp

```diff
@@ -293,6 +308,11 @@ class MQTTDisconnected : public StateMachine {
       }      
     }
   }
+  
+  void react(MQTTDisconnectedEvent const &) {
```

Comment:
This is not need when nothing is done I believe

Reply:
These are quite important as the default behavior as defined in StateMachine class is to do something, see https://github.com/Romkabouter/ESP32-Rhasspy-Satellite/blob/00a58cfa8dc2096a45351aa16fbb7af25de17c4e/PlatformIO/src/StateMachine.hpp#L19 

Not doing anything is here the right choice as opposed to switch to MQTTDisconnected state (which is the default and good for all cases except for the ones with the "empty" code).

Reply:
Switching to MQTTDisconnected state before wifi is connected will not work. However, I2S Task will try to do exactly this as soon as it is started. Initially I use fsm::is_in_state<...>() to prevent unwanted switching. But this way it is much safer: Each state takes care of ignoring state change events not applicable to its current condition.

Reply:
Sorry, you are correct. I forgot that I created default behavior a while ago......